apps/server/src/modules/board/uc/board.uc.spec.ts

```diff
@@ -232,4 +234,16 @@ describe(BoardUc.name, () => {
 			});
 		});
 	});
+
+	describe('copyBoard', () => {
+		it('should call the service to copy the board', async () => {
+			const { user, board } = setup();
+
+			await uc.copyBoard(user.id, board.id);
+
+			expect(columnBoardCopyService.copyColumnBoard).toHaveBeenCalledWith(
+				expect.objectContaining({ userId: user.id, originalColumnBoardId: board.id })
+			);
+		});
```

Comment:
we have good coverage from api tests, but to me it looks like we should unit test the other parts of the copyBoard too, like making sure we are calling authorization to check permission and so on